public/configs/someMarketData.json

```diff
@@ -0,0 +1,1890 @@
+      "delayBlocks": 3600
+    },
+    "initialDeposit": {
+      "denom": "adydx",
+      "amount": "10000000000000000000000"
+    },
+    "meta": {
+      "assetName": "Waffles Davincij15's Cat",
+      "referencePrice": 0.0185995037907017,
+      "cmcId": 31442
+    }
+  },
+  "FALX": {
+    "title": "Add FALX-USD perpetual market",
+    "summary": "Add the x/prices, x/perpetuals and x/clob parameters needed for a FALX-USD perpetual market. Create the market in INITIALIZING status and transition it to ACTIVE status after 3600 blocks. Added via the new market widget.",
+    "params": {
+      "id": 636,
+      "ticker": "FALX-USD",
+      "marketType": "PERPETUAL_MARKET_TYPE_ISOLATED",
+      "priceExponent": -12,
+      "minPriceChange": 800,
+      "minExchanges": 1,
+      "exchangeConfigJson": [
+        {
+          "exchangeName": "Raydium",
+          "ticker": "FALX-SOL-6TtniYJPdHJ764d3rUvk8SokmzyZMYCgqUjSgUAnjma2-9-2eH2DVXQvs5qWwDQjgiSPsdZ19KQsj84RKRfCeorsGem-9-3jf9f9VJdUXQQha6nHJkZVxqBTW5oJUNHYuKLyfDDuMM-2hPp2aKd6T6HZmMQW2LkqH7R1wLZDjzZ1bZjhj5nrhrV",
+          "adjustByMarket": "SOL-USD",
+          "invert": false
+        }
+      ],
+      "liquidityTier": 2,
+      "atomicResolution": -3,
+      "quantumConversionExponent": -9,
+      "defaultFundingPpm": 0,
+      "stepBaseQuantums": 1000000,
+      "subticksPerTick": 1000000,
+      "delayBlocks": 3600
+    },
+    "initialDeposit": {
+      "denom": "adydx",
+      "amount": "10000000000000000000000"
+    },
+    "meta": {
+      "assetName": "FalconX",
+      "referencePrice": 0.0010943301005112,
+      "cmcId": 31384
+    }
+  },
+  "KHAI": {
+    "title": "Add KHAI-USD perpetual market",
+    "summary": "Add the x/prices, x/perpetuals and x/clob parameters needed for a KHAI-USD perpetual market. Create the market in INITIALIZING status and transition it to ACTIVE status after 3600 blocks. Added via the new market widget.",
+    "params": {
+      "id": 637,
+      "ticker": "KHAI-USD",
+      "marketType": "PERPETUAL_MARKET_TYPE_ISOLATED",
+      "priceExponent": -10,
+      "minPriceChange": 800,
+      "minExchanges": 1,
+      "exchangeConfigJson": [
+        {
+          "exchangeName": "Raydium",
+          "ticker": "KHAI-SOL-6g4rhxVTrN6SrtNvimq4QiU8yA5XScvwL6wxaMkegrtJ-8-7p2PrGMyeetNRqTKFraL7eYo2TbU3apWz6vfqrZFiPcG-9-2DaRg4UycKL9GSVfARBDrcensb89WD5PyyFX9NrMunLc-ECbK6PSMZ5yQaUYBocsXaVrax2fWADw2ijTqLGPtt9sC",
+          "adjustByMarket": "SOL-USD",
+          "invert": false
+        }
+      ],
+      "liquidityTier": 2,
+      "atomicResolution": -5,
+      "quantumConversionExponent": -9,
+      "defaultFundingPpm": 0,
+      "stepBaseQuantums": 1000000,
+      "subticksPerTick": 1000000,
+      "delayBlocks": 3600
+    },
+    "initialDeposit": {
+      "denom": "adydx",
+      "amount": "10000000000000000000000"
+    },
+    "meta": {
+      "assetName": "Kitten Haimer",
+      "referencePrice": 0.500492,
+      "cmcId": 30948
+    }
+  },
+  "BOOMER": {
```

Comment:
Looks like this pair has v [low](https://dexscreener.com/solana/31qgknu26ay21vkwgcvau2nzech56f9crktpxtmylyme) liquidity

Reply:
I suggest we remove